cursorless-nx/libs/cheatsheet/src/lib/data/sampleSpokenFormInfos/defaults.json

```diff
@@ -5,252 +5,266 @@
 			"id": "actions",
 			"items": [
 				{
-					"id": "scrollToBottom",
```

Comment:
alphabetised / added things which had been forgotten from previous updates.  Remember this file is just for convenience during local cheatsheet development